wp-content/themes/sirf/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sirf-theme",
+  "version": "1.0.0",
+  "description": "Theme for sirf-onlien.org",
```

Comment:
typoed 'online' but that's low priority to fix.

Reply:
Fixed in https://github.com/INN/umbrella-sirf/pull/7/commits/5430fc9002c9cecf65ead323e69437ef9e58eecc